java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1825,6 +1850,14 @@ public long getNativeView() {
    */
   private static native long substringLocate(long columnView, long substringScalar, int start, int end);
 
+  /**
+   * Native method to calculate substring from a given string column. 0 indexing.
+   * @param columnView native handle of the cudf::column_view being operated on.
+   * @param start      first character index to begin the substring(inclusive).
+   * @param end        last character index to stop the substring(exclusive).
+   */
+  private static native long subString(long columnView, int start, int end);
```

Comment:
Nit: Should we not camelCase this method-name? See above: `substringLocate()`.

Reply:
I had made it camelCase for this method. Since it was just substring, made it as subString

Reply:
Sorry for bring nit picky but shouldn't this be just `substring`

Reply:
I was a bit confused if it needs to be `substring` . Will update it. Thanks @mythrocks and @kuhushukla.